TicketManagement/src/main/java/com/example/TicketManagement/dto/OrderDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.example.TicketManagement.dto;
+
+import com.example.TicketManagement.model.Customer;
+import com.example.TicketManagement.model.TicketCategory;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderDTO {
```

Comment:
Add here the eventId as given in the sample requests

Reply:
Done